functions.php

```diff
@@ -518,9 +518,9 @@ function twentytwenty_get_post_meta( $post_id = null, $location = 'single-top' )
 
 		$page_template = get_page_template_slug( $post_id );
 
-		// Check that the post type should be able to output post meta
-		$allowed_post_types = apply_filters( 'twentytwenty_allowed_post_types_for_meta_output', array( 'post' ) );
-		if ( ! in_array( get_post_type( $post_id ), $allowed_post_types ) ) {
+		// Check whether the post type is allowed to output post meta
```

Comment:
Inline comments must end in full-stops, exclamation marks, or question marks.
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#5-inline-comments